### PR TITLE
Add rubygem-foreman_statistics package

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Add foreman_statistics plugin.
Installer: https://github.com/theforeman/puppet-foreman/pull/855

### CODE REVIEW

File: packages/plugins/rubygem-foreman_statistics/rubygem-foreman_statistics.spec [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
+# template: foreman_plugin
```
Comment (tbrisker) [inline packages/plugins/rubygem-foreman_statistics/rubygem-foreman_statistics.spec:5]: This is likely what koji is mad about

Comment (ezr-ondrej) [reply packages/plugins/rubygem-foreman_statistics/rubygem-foreman_statistics.spec]: could be :thinking: :man_facepalming:

**Comment by tbrisker** [comment]
error: File not found: /builddir/build/BUILDROOT/tfm-rubygem-foreman_statistics-0.1.0-1.fm2_2.el7.noarch/opt/theforeman/tfm/root/usr/share/gems/gems/foreman_statistics-0.1.0/public/apipie-cache/plugin/foreman_statistics

**Comment by ezr-ondrej** [comment]
:green_apple: :tada:

**Comment by ezr-ondrej** [comment]
Lets just release 0.1.1 right away.

**Comment by ezr-ondrej** [comment]
I've just squashed (even the package history)

File: comps/comps-foreman-plugins-rhel7.xml [hunk 2]
```diff
@@ -82,6 +82,7 @@
       <packagereq type="default">tfm-rubygem-foreman_setup</packagereq>
       <packagereq type="default">tfm-rubygem-foreman_snapshot_management</packagereq>
       <packagereq type="default">tfm-rubygem-foreman_spacewalk</packagereq>
+      <packagereq type="default">tfm-rubygem-foreman_statistics</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml:85]: can you please add similar changes (minus the `tfm-` prefix) to `comps-foreman-plugins-el8.xml` please? You can use `add_to_comps.rb` for that.

File: rel-eng/tito.props [hunk 3]
```diff
@@ -921,6 +921,7 @@ whitelist = nodejs-babel-plugin-module-resolver
   rubygem-foreman_setup
   rubygem-foreman_snapshot_management
   rubygem-foreman_spacewalk
+  rubygem-foreman_statistics
```
Comment (evgeni) [inline rel-eng/tito.props:924]: could you add the same to the `foreman-plugins-nightly-el8` section too please?

File: comps/comps-foreman-plugins-rhel7.xml (hunk 2 above)
Comment (ezr-ondrej) [reply comps/comps-foreman-plugins-rhel7.xml:85]: sure, would be nice to add it to the `add_gem_package.sh` script as well :thinking: :)

File: rel-eng/tito.props (hunk 3 above)
Comment (ezr-ondrej) [reply rel-eng/tito.props:925]: your wish is my command :upside_down_face:

File: comps/comps-foreman-plugins-rhel7.xml (hunk 2 above)
Comment (evgeni) [reply comps/comps-foreman-plugins-rhel7.xml:85]: yeah. it's a bit broken for the two-os usecase we have right now.

**Pull request merged**